datahub-web-react/src/app/identity/user/UserListItem.tsx

```diff
@@ -119,6 +191,19 @@ export default function UserListItem({ user, canManageUserCredentials, onDelete
                     <MenuIcon fontSize={20} />
                 </Dropdown>
             </ButtonGroup>
+            <ViewAssignRoleModal
```

Comment:
Why is there a modal for assigning a role here? I thought it's just a select on the user themself

Reply:
Like there's a role select just above - what is this extra code for?

Reply:
Refactored into popconfirm per offline discussion!